src/storage/conversion/RdfToQuadConverter.ts

```diff
@@ -39,8 +39,8 @@ export class RdfToQuadConverter extends TypedRepresentationConverter {
 
     // Wrap the stream such that errors are transformed
     // (Node 10 requires both writableObjectMode and readableObjectMode)
-    const data = new PassThrough({ writableObjectMode: true, readableObjectMode: true });
-    pipeStreamsAndErrors(rawQuads, data, (error): Error => new UnsupportedHttpError(error.message));
+    const pass = new PassThrough({ writableObjectMode: true, readableObjectMode: true });
+    const data = pipeStreamsAndErrors(rawQuads, pass, (error): Error => new UnsupportedHttpError(error.message));
```

Comment:
Some test did not catch something here.

Reply:
I changed it to use the function output for consistency (and in case the function ever gets changed to use something else, which was the original plan with `pipeline), but in practice it currently returns the second stream, so I missed this one.

Reply:
Going through this again I realized I misinterpreted your comment and actually have no idea what the issue is here? 😄

Reply:
Never mind, I was mistaken.